addons/web_editor/static/src/js/editor/odoo-editor/test/spec/insertHTML.test.js

```diff
@@ -90,6 +93,20 @@ describe('insert HTML', () => {
                 contentAfter: '<pre>abcdef[]<br>ghi</pre>',
             });
         });
+        it('should unpack elements from a lone paragraph (Knowledge check)', async () => {
```

Comment:
It's kind of weird to explicitely reference knowledge in an editor community test.